translations/base-pl.yaml

```diff
@@ -72,17 +72,19 @@ global:
     error: Wystąpił błąd
 
     # How big numbers are rendered, e.g. "10,000"
-    thousandsDivider: "."
+    thousandsDivider: " "
 
     # The suffix for large numbers, e.g. 1.3k, 400.2M, etc.
+	# Translator note: We don't use SI size units for common speak, but if you want to keep it SI
```

Comment:
Could you fix the spaces there? I think there are 5 spaces here instead of 4

Reply:
On my way!